addons/product_expiry/tests/test_generate_serial_numbers.py

```diff
@@ -0,0 +1,200 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+from freezegun import freeze_time
+
+from odoo.addons.stock.tests.test_generate_serial_numbers import StockGenerateCommon
+from odoo.tests.common import Form
+from odoo.tools.misc import get_lang
+
+
+class TestStockLot(StockGenerateCommon):
+    def test_set_multiple_lot_name_with_expiration_date_01(self):
+        """ In a move line's `lot_name` field, pastes a list of lots and expiration dates.
+        Checks the values are correctly interpreted and the expiration dates are correctly created
+        depending of the user lang's date format.
+        """
+        product_lot = self.env['product.product'].create({
+            'name': 'Tracked by Lot Numbers',
+            'tracking': 'lot',
+            'type': 'product',
+        })
```

Comment:
How do these tests work without `use_expiration_date=True`? :exploding_head:

Reply:
It works because the field `expiration_date` exist once `product_expiry` is installed, no matter if the product actually uses dates, but you're right, it shouldn't work in this case :grimacing: 
It's better to ignore expiration date in this case, so I'll fix that, thank :+1: